config.default

```diff
@@ -0,0 +1,75 @@
+{
+  "common": {
+    "sync_on_start": false
```

Comment:
When one may want to disable the option?

Reply:
When someone doesn't want to sync the repositories at startup). I think this is possible when debugging.

Reply:
Will not it ruin guarantees the service gives? Why not to sync at debugging too?

Reply:
(Anyway, now we'll the only admins of the service, so all consequences are our. My humble opinion is that ideally we should have a good reason to provide the second, relaxed set of guarantees.)

Reply:
Removed the "sync_on_start" parameter from configuration file.